demisto_sdk/commands/common/tests/dependencies_test.py

```diff
@@ -111,18 +111,19 @@ def make_indicator_type_depend_on_script_enhancement(indicator_type: JSONBased,
         script_id = script.yml.read_dict().get('commonfields').get('id')
         indicator_type.update({'enhancementScriptNames': [script_id]})
 
-    @staticmethod
-    def make_indicator_type_depend_on_integration(indicator_type: JSONBased, integration: Integration):
-        indicator_type.update({'reputationCommand': 'ip'})
-        integration.yml.update(
-            {
-                'script': {
-                    'commands': [{
-                        'name': 'ip'
-                    }]
-                }
-            }
-        )
+    # Optional dependency
```

Comment:
why was that removed?

Reply:
At the beginning I thought that it wont be included since its an optional dependency.
Put it back